docs/explanations.rst

```diff
@@ -59,6 +59,22 @@ Messaging
    
    Messages... <explanations/index_messages>
 
+CVE Commands
+=============
+
+Here we'll show you the information related to CVE related commands.
+
+On any of the listed commands, we consider a CVE vulnerability as **fixable**,
+if and only if, at least **one** affected installed package associated with it can
+be fixed. That means that a vulnerability is unfixable if no affected installed packages
+related to the vulnerability can be fixed at the moment.
+
+..  toctree::
+    :maxdepth: 1
+
+    explanations/how_to_interpret_output_of_pro_cves.md
+    explanations/how_to_interpret_output_of_pro_cve.md
```

Comment:
I don't think it makes sense to have two separate explanation pages for this - can we combine these into a single how-to that can disambiguate between the two commands? They're so similarly named that I worry having the two presented side-by-side like this is going to confuse users.

If that's out of scope, I would at least recommend not calling them "how to interpret...", but rather naming the pages according to the intended use of the command.